Health App/Home/AddOrUpdateHealthConcernVC.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  AddOrUpdateHealthConcernVC.swift
+//  Health App
+//
+//  Created by Koushik Dutta on 07/10/17.
+//  Copyright © 2017 Virtual Elf LLC. All rights reserved.
+//
+
+import UIKit
+import DataLayer
+class AddOrUpdateHealthConcernVC: UIViewController {
+    
+    @IBOutlet weak var statusPickerView: UIPickerView! = UIPickerView()
+    @IBOutlet weak var titleTextField: UITextField!
+    @IBOutlet weak var statusTextField: UITextField!
+    @IBOutlet weak var noteTextView: UITextView!
+    @IBOutlet weak var outerView: UIView!
+    weak var healthConcern : HealthConcern?
+    let statusOptions = ["In Control","Not In Control","Resolved"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        customizedTextView()
+        self.statusTextField.inputView = statusPickerView
+        if let title = healthConcern?.title, !title.isEmpty {
```

Comment:
`Delete` button should be enabled only if healthConcern is not nil
else disable

Reply:
same for add Action Plan

Reply:
I'm not sure about action plan yet... don't make that change now